frontend/components/admin/patient/patient-modal.module.css

```diff
@@ -0,0 +1,60 @@
+.modalHeader {
+    font-family: Poppins;
+    font-size: 1.5em;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 54px;
+    letter-spacing: 0em;
+    text-align: center;
+}
+
+.selectionFont {
+    font-family: Poppins;
+    font-size: 1.2em;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 42px;
+    letter-spacing: 0em;
+    text-align: left;
+
+    color: #444444;
+}
+
+.roleFont {
+    font-family: Poppins;
```

Comment:
there's no need to include `font-family: Poppins;`, afaik it's already set as a default in our global css file